adapters/inmobi/inmobi.go

```diff
@@ -0,0 +1,97 @@
+package inmobi
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type InmobiAdapter struct {
```

Comment:
I recommend the adapter names to match the capitalization of your brand. Please consider:
InmobiAdapter -> InMobiAdapter
NewInmobiAdapter -> NewInMobiAdapter

Reply:
Should be resolved where applicable.